openqabot/smeltsync.py

```diff
@@ -35,35 +35,25 @@ def __call__(self) -> int:
         return ret
 
     @staticmethod
-    def _review_rrequest(requestSet):
-        if not requestSet:
+    def _review_rrequest(request_set):
+        if not request_set:
             return None
-        else:
-            rr = sorted(requestSet, key=itemgetter("requestId"), reverse=True)[0]
-            if rr["status"]["name"] in ("new", "review", "accepted", "revoked"):
-                return rr
-            else:
-                return None
+        rr = sorted(request_set, key=itemgetter("requestId"), reverse=True)[0]
+        if rr["status"]["name"] in ("new", "review", "accepted", "revoked"):
+            return rr
+        return None
 
     @staticmethod
     def _is_inreview(rr_number) -> bool:
         if rr_number["reviewSet"]:
-            if rr_number["status"]["name"] == "review":
-                return True
-            else:
-                return False
-        else:
-            return False
+            return rr_number["status"]["name"] == "review"
```

Comment:
[if/else] To be carefully reviewed

Reply:
change is fine. an equality check directly returns a boolean result so it's equivalent to returning True/False and the change is good.